src/assets/locales/de/translations.json

```diff
@@ -35,7 +35,7 @@
   "career.privacy-policy": "<0>Hiermit bestätige ich, dass ich die <1>Datenschutzerklärung</1> zur Kenntnis genommen habe. <4>*</4></0>",
   "career.introduction.headline": "Arbeite mit uns",
   "career.introduction.kicker": "Karriere bei Satellytes",
-  "career.introduction.paragraphs.0": "Wir sind Satellytes, ein Team aus sehr erfahrenen Entwicklern und einigen Designern. Wir bringen großen Unternehmen nachhaltige, barrierefreie, verlässliche und moderne Lösungen rund ums Web. Wenn du Interesse hast, bei uns mitzumachen, dann schau dir gleich unserer offenen Positionen an.",
+  "career.introduction.paragraphs.0": "Wir sind Satellytes, ein Team aus sehr erfahrenen Software Ingenieuren und einigen Designern. Wir bringen großen Unternehmen nachhaltige, barrierefreie, verlässliche und moderne Lösungen rund ums Web. Wenn du Interesse hast, bei uns mitzumachen, dann schau dir gleich unserer offenen Positionen an.",
```

Comment:
I am not quite sure, but I think this sounds a bit weird, doesn't it? It seems that we have "(sehr erfahrene Software Ingenieure)" and on the other hand "(einige Designer)". What do you think about leaving out the "einige" to have "(sehr erfahrene Software Ingenieure und Designer)"?